### PR TITLE
Alex feature user profile

### PR DESCRIPTION
**Pull request by AlexanderHMagno**
# Update Profile Screen

* Reusing React Context for retrieving user information.
* Query list of interest
* Mutation. Update User information
* Save in the database.

## Issues 
* Styling, the new elements have modified the aspect of the profile user, is necessary to style it again. 
* The design only has one space for the name, I have added a second input field for the last name. 
* Updating the information in the Database takes time. 

## Next Steps 
* Refract the style.
* Refract code
* Terminate the mutation, currently only the profile information has been updated, interest and favourite ways have to be updated too.

### CODE REVIEW

File: client/js/screens/UserProfile/helpers/source.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+let source = {
```
Comment (siddharthparmar7) [inline client/js/screens/UserProfile/helpers/source.js]: If this is a helper file, it should be in the helpers dir. Please make the changes.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

File: client/js/screens/UserProfile/helpers/source.js (hunk 1 above)
Comment (AlexanderHMagno) [reply client/js/screens/UserProfile/helpers/source.js]: I have moved the folder to the correct position.

File: client/js/screens/UserProfile/components/FaveWays.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+class FaveWays extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      style: styles.icon,
+      image: true,
+    };
+  }
+  componentWillMount() {
```
Comment (siddharthparmar7) [inline client/js/screens/UserProfile/components/FaveWays.js]: componentWillMount lifecycle method is deprecated. Please use componentDidMount instead.

File: client/js/screens/UserProfile/components/source.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+let source = {
```
Comment (siddharthparmar7) [inline client/js/screens/UserProfile/components/source.js]: components dir should not be inside a screen dir. Please move it to the components dir and if it is a helper file then move it to the helpers dir. Thank you.

File: client/js/screens/UserProfile/styles.js [hunk 4]
```diff
@@ -49,54 +52,71 @@ const styles = StyleSheet.create({
...
     flexDirection: 'row',
     justifyContent: 'space-around',
     alignItems: 'center',
-    marginVertical: '2%',
-    textAlign: 'center',
-    height: 'auto',
+    alignContent: 'center',
+    marginTop: '3%',
+    textAlign: 'justify',
+    // height: 'auto',
```
Comment (siddharthparmar7) [inline client/js/screens/UserProfile/styles.js]: remove commented code.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the changes.

File: client/js/screens/UserProfile/components/FaveWays.js (hunk 2 above)
Comment (AlexanderHMagno) [reply client/js/screens/UserProfile/components/FaveWays.js]: Updated

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**